recipe/meta.yaml

```diff
@@ -11,29 +11,35 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
-  noarch: python
-  script: python setup.py install --prefix ${PREFIX}
+  number: 0
+  skip: True  # [win or osx or py<=27]
 
 requirements:
   build:
+    - setuptools
```

Comment:
@stompsjo  Could you check if this is actually required?
I am having doubt...

Reply:
@bam241 I removed it, will wait to see if the builds pass.